lambda_functions/source/cw-helper/organizations.py

```diff
@@ -37,9 +41,12 @@ def lambda_handler(event, context):
         context: runtime information
     """
     try:
-        data = get_org_id()
-        cfnresponse.send(event, context, cfnresponse.SUCCESS, data, data["OrganizationId"])
+        organization_id, eventbus_account_ou = get_org_id()
+        data_dict = {}
```

Comment:
define data_dict on the top. Failure in get_org_id() would result in failure of `cfnresponse.send` call.